collections/_article/godot-xr-update-oct-2024.md

```diff
@@ -0,0 +1,136 @@
+
+### Improved WebXR support
+
+WebXR support in Godot is seeing continuous improvement, most notably the addition of hand tracking support, but also support for MSAA and a number of smaller bug fixes and improvements.
+Our WebXR implementation now also supports AR applications.
+
+### Improved OpenGL support
+
+While technically not specific to XR, several improvements to the compatibility renderer directly impact the ability to create XR experiences using Godot.
+
+The new MSAA implementation in the compatibility renderer has support for OpenGL extensions specifically targeting TBDR GPUs found on modern standalone headsets.
+When available enabling MSAA 2x or 4x has a very low cost while providing worth while improvement in the visuals.
+
+The new lightmapping capabilities of the compatibility renderer, where the Vulkan implementation is leveraged to bake the lightmaps, provides a viable solution for improved lighting on standalone XR hardware.
+
+### Improved foveated rendering support in the Vulkan engine
+
+The compatibility renderer uses OpenXRs foveated rendering support directly but this comes with various limitations and is not applicable outside of OpenXR.
+
+In the Vulkan renderer we have had Variable Rate Shading support for a while which allows foveated rendering to be applied in Godot without the same drawbacks.
+
+In the latest release various improvements have been applied that provide more control over the density map used and allow for eye tracking to impact the density map and ensure quality rendering at the users focal point.
+
+## Collaboration between W4 Games & Meta 
+
+The past year saw W4 Games and Meta join forces to improve Godot's support for the Quest line of devices.
+
+### Core functionality improvements
+
+The team at W4 has worked closely together with several core Godot XR contributors to improve the core functionality in Godot, improving the experience not just for Meta Quest users, but also for other XR systems. Several of the items mentioned in the previous sections benefited greatly from the help provided here.
+
+These improvements also include various rendering optimisations and fixes.
+
+### Enhanced passthrough and hand-tracking support
+
+On top of the standardisation of passthrough and hand-tracking in Godot, support for many extra features Meta provides was implemented in the Godot OpenXR Vendors plugin.
+
+For passthrough, this includes applying various filters to the passthrough imagery, like adjusting the brightness, contrast and saturation, mapping the colors to other values, or even projecting passthrough onto custom geometry, in order to bring only specific objects from the user's physics environment into an otherwise VR experience. See the Meta Passthrough tutorial for more information.
```

Comment:
```suggestion
For passthrough, this includes applying various filters to the passthrough imagery, like adjusting the brightness, contrast and saturation, mapping the colors to other values, or even projecting passthrough onto custom geometry, in order to bring only specific objects from the user's physics environment into an otherwise VR experience. See the [Meta Passthrough tutorial](https://godotvr.github.io/godot_openxr_vendors/manual/meta/passthrough.html) for more information.
```